cluster/manifests/z-karpenter/deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       #   node.kubernetes.io/role: master
       containers:
         - name: controller
-          image: "public.ecr.aws/karpenter/controller:v0.16.0@sha256:15cb6fe00279442b752beaf642dc9d6a3a2a9570d6da77b9bd7681e4c61b0958"
+          image: "public.ecr.aws/karpenter/controller:v0.16.1@sha256:ed1f97bc2cd67e63b45546955cf763890182f1f97586fca5e6cbd25f175b9c56"
```

Comment:
Note, we also need/should update the webhook container image which is the second container in this deployment pod :)